### PR TITLE
Feat/alert variants

### PR DESCRIPTION
**Pull request by carloshrabelo**
## Infos

[Task](https://juntossomosmais.monday.com/boards/346135329/views/4459666/pulses/750798548)

#### What is being delivered?

Variações para o componente de Alert

#### What impacts?

Implementações futuras dos componentes de Alert

#### Reversal plan

Reverter branch

#### Evidences

Screen Shot 2020-09-18 at 18 32 36
Screen Shot 2020-09-18 at 18 32 27

## DoD checklist

- [x] Related code finished
- [X] Documentation updated (if exists)
- [x] Unit tests written and passing
- [x] Any configuration or build changes documented
- [x] Project builds without errors
- [x] Lint errors fixed

### CODE REVIEW

File: packages/react-ds/src/components/Alert/index.tsx [hunk 1]
```diff
@@ -58,16 +54,21 @@ export const Alert: React.FunctionComponent<
   closable,
   onClose,
   type = '',
+  transparent = false,
```
Comment (felipefialho) [inline packages/react-ds/src/components/Alert/index.tsx:57]: maybe we should use a prop called `variant` to provide more options to use together alert

````
variant: 'transparent' | 'filled'
```

File: packages/styles/components/Alert.module.scss [hunk 2]
```diff
@@ -3,28 +3,103 @@
 $colors: 'primary' var(--primary-color), 'secondary' var(--secondary-color),
   'success' var(--success-color), 'danger' var(--danger-color);
 
-$iconSize: var(--space-sm);
+$icon-sz-sm: 12px;
+$icon-sz: var(--space-sm);
+$icon-sz-lg: 24px;
```
Comment (felipefialho) [inline packages/styles/components/Alert.module.scss:8]: I took a delay to understand what `sz` means, maybe is better use a explicit name like `size`.

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: packages/styles/components/Alert.module.scss (hunk 2 above)
Comment (carloshrabelo) [reply packages/styles/components/Alert.module.scss]: made on 660a5446171c07d6d9ca3645380979bdd937afcf

File: packages/react-ds/src/components/Alert/index.tsx (hunk 1 above)
Comment (carloshrabelo) [reply packages/react-ds/src/components/Alert/index.tsx]: made on 660a5446171c07d6d9ca3645380979bdd937afcf

**Review by felipefialho — APPROVED** [review]
Nice!

**Review by felipefialho — APPROVED** [review]

**Pull request merged**